acsoo/templates/project/+project.name+/acsoo.cfg.bob

```diff
@@ -1,4 +0,0 @@
-[acsoo]
-trigram={{{ project.trigram }}}
-series={{{ odoo.series }}}
-version=1.0.0
```

Comment:
Where can we put rules for acsoo checklog ?

Reply:
still in `acsoo.cfg` (until we replace acsoo checklog by a standalone tool).

Maybe the template should generate a template with an empty `checklog` section with an commented-out example.